params-plugin/tests/pll/pll.tcl

```diff
@@ -0,0 +1,32 @@
+yosys -import
+plugin -i xdc
+plugin -i params
+#Import the commands from the plugins to the tcl interpreter
+yosys -import
+
+read_verilog -specify -lib -D_EXPLICIT_CARRY +/xilinx/cells_sim.v
+read_verilog -lib +/xilinx/cells_xtra.v
+hierarchy -check -auto-top
+set phase [getparam CLKOUT2_PHASE top/PLLE2_ADV_0 top/PLLE2_ADV]
```

Comment:
This test doesn't actually verify that phase returns:
 - Correct number of values
 - That the values are correct?